pkgs/applications/editors/emacs-modes/emacspeak/default.nix

```
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, rsync, gnused, emacs, tcl, tclx, espeak-ng }:
+
+stdenv.mkDerivation rec {
+  pname = "emacspeak";
+  version = "51.0";
+
+  src = fetchFromGitHub {
+    owner = "tvraman";
+    repo = "emacspeak";
+    rev = version;
+    sha256 = "06azq2566pj4pnyawqm9387ybp3wr1pr6828hhdxhki4dpn9pj8g";
+  };
+
+  nativeBuildInputs = [ rsync gnused ];
```

sed is already in stdenv.

```suggestion
  nativeBuildInputs = [ rsync ];
```